### ISSUE TITLE
Rfox tx's need parsing for safe wallet transactions 

### ISSUE THREAD

**Issue by MBMaria** [shapeshift/web#7765]
### Overview

When approving/withdrawing or changing address for rfox while connected to a safe wallet, the tx's are listed as transaction instead of what they actually are

### References and additional details

![image](https://github.com/user-attachments/assets/dc3a0c08-7b6d-4662-8c3b-6da87b2cbb5c)

### Acceptance Criteria

Txs should be listed accordingly 

### Ownership

- [X] If my bounty needs engineering or needs product I have added the respective labels on the right
- [X] As the sponsor of this bounty I will review the changes in a preview environment (ops/product) or review the PR (engineering)

### Bounty Hunters

- [Join our discord](https://discord.gg/shapeshift)
- Include an expected timeline for you to complete work in the work plan when you apply for this bounty!
- Please refer to [this link](https://shapeshiftdao.gitbook.io/getting-started/how-to-contribute/bounties#resources-for-new-contributors-and-bounty-hunters) for some basic info
- Please ***do not*** start work on this issue until you are approved in Gitcoin.

**Comment by gomesalexandre** [comment]
Closing for the time being @MBMaria  - to be maybe continued after https://github.com/shapeshift/web/issues/8956 goes in.
tl;dr is this is not isolated to RFOX and any SAFE Tx is potentially affected.

We *absolutely* could decode the on-chain Tx from the SAFE Tx hash, which would get very heavy very fast.
Easy if we're only displaying incoming Txs from ws (read: new Txs), a perf nightmare if we were to do so for all historical Txs since SAFE Tx parsing requires:

1. Detecting whether or not a Tx is a SAFE Tx (one network request)
2. If it is, fetching the actual underlying Tx (another request)
